test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -1547,7 +1556,7 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
       if (status != SAI_STATUS_SUCCESS) {
           return status;
       }
-      status = acl_api->delete_acl_table(acl_table_id);
+      status = acl_api->remove_acl_table(acl_table_id);
```

Comment:
no the latest sai header, the sonic branch. https://github.com/opencomputeproject/SAI/tree/sonic